official/utils/flags/_performance.py

```diff
@@ -34,6 +34,10 @@
 
 
 def get_tf_dtype(flags_obj):
```

Comment:
where all is this API used? Are all those codepaths also using the new API to support fp16?

Reply:
This is used in the estimator and keras models. Since --fp16_implementation is only defined for the estimator model, this change has no impact on the keras model.